src/AutoRest.CSharp/Generation/Writers/RestClientWriter.cs

```diff
@@ -246,15 +246,19 @@ private void WriteRequestCreation(CodeWriter writer, RestClientMethod clientMeth
                         break;
                     case UrlEncodedBody urlEncodedRequestBody:
                         WriteHeaders(writer, clientMethod, request, content: true);
-                        writer.Append($"var contentBuilder = new Azure.Core.FormUrlEncodedContent.Builder ();\n");
+                        writer.Append($"var content = new Azure.Core.FormUrlEncodedContent ();\n");
```

Comment:
I know I said you done need to use CodeWriterDeclaration but I think you do in case there is a parameter named content.
See multipart for an example:

```C#
 var multipartContent = new CodeWriterDeclaration("content");
writer.Line($"var {multipartContent:D} ... ");
...

writer.Append($"{multipartContent}.Add(...)");
```

This way we'll get the variable name deduplication for free.